envsec/internal/jetcloud/project.go

```diff
@@ -0,0 +1,63 @@
+package jetcloud
+
+import (
+	"encoding/json"
+	"os"
+	"path/filepath"
+
+	"github.com/pkg/errors"
+	"go.jetpack.io/envsec/internal/auth"
+)
+
+const dirName = ".jetpack"
```

Comment:
Given that we're going to change the company name, will this mean we'll have to migrate the data in the directory?